README.md

```diff
@@ -1 +1,92 @@
-# connector-soap
\ No newline at end of file
+# Ballerina Soap connector
```

Comment:
Follow a convention, either capitalize all the first letters of a word. or only the first in the heading